src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,70 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  padding: 10px;
+  width: $calendar-width;
+
+  $start-day: (
+    'mon': 0,
+    'tue': 1,
+    'wed': 2,
+    'thu': 3,
+    'fri': 4,
+    'sat': 5,
+    'sun': 6,
+  );
```

Comment:
you can move it to variables file too

Reply:
i will do it
another problem: the task specifies the color #FFBFCB for the hovered cells, but the test identifies it as incorrect